app/controllers/api_entreprise/restricted_token_magic_links_controller.rb

```diff
@@ -22,6 +17,15 @@ def create
 
   private
 
+  def organizer
+    @organizer = Token::DeliverMagicLinkToEmail.call(
```

Comment:
```suggestion
    @organizer ||= Token::DeliverMagicLinkToEmail.call(
```

sinon ça sert à rien d'affecter la variable (sauf si on la manipule derrière mais peu conventionnel)